contracts/Makefile

```diff
@@ -14,8 +14,7 @@ build: version ## Build contracts.
 
 CORE_CONTRACTS := OmniPortal FeeOracleV1 Create3 TransparentUpgradeableProxy \
 			Staking Slashing OmniBridgeL1 OmniBridgeNative Omni WOmni \
-			PortalRegistry AllocPredeploys PingPong \
-			core/src/deploy/ProxyAdmin.sol:ProxyAdmin \
+			PortalRegistry AllocPredeploys PingPong ProxyAdmin \
```

Comment:
how/where do we specify that `ProxyAdmin` is v5 instead of v4

Reply:
in core directory we only have v5 bindings